db/schema.rb

```diff
@@ -86,7 +86,7 @@
     t.uuid "accountable_id"
     t.decimal "balance", precision: 19, scale: 4, default: "0.0"
     t.string "currency", default: "USD"
-    t.virtual "classification", type: :string, as: "\nCASE\n    WHEN ((accountable_type)::text = ANY (ARRAY[('Account::Loan'::character varying)::text, ('Account::Credit'::character varying)::text, ('Account::OtherLiability'::character varying)::text])) THEN 'liability'::text\n    ELSE 'asset'::text\nEND", stored: true
+    t.virtual "classification", type: :string, as: "\nCASE\n    WHEN ((accountable_type)::text = ANY ((ARRAY['Account::Loan'::character varying, 'Account::Credit'::character varying, 'Account::OtherLiability'::character varying])::text[])) THEN 'liability'::text\n    ELSE 'asset'::text\nEND", stored: true
     t.boolean "is_active", default: true, null: false
```

Comment:
Which Postgres version are you running locally?  Think the schema dump may have just interpreted this differently

Reply:
v 13.13. I could not find why it was interpreted differently.

Reply:
Gotcha.  I could be wrong, but I _think_ if you update to PG 14+ you won't run into this

Reply:
Thanks. I'll try to update pg.